en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+# Duo-Tailsitter VTOL Configuration
+
+This is the configuration tuning documentation for a "Duo-Tailsitter VTOL".
+
+A tailsitter is essentially a fixed wing vehicle that takes off in a pitched up configuration (essentially sitting on its tail, hence the name) and then transitions to fixed wing flight by pitch down by 90 degrees to achieve the fixed wing configuration.
+The Duo-Tailsitter variant is so-named because it has two motors (a "quad" tailsitter would have four).
+
+Like other VTOLs, the Duo-Tailsitter has two modes of operation: multicopter (MC) mode and fixed wing (FW) mode.
+In MC mode, lift and roll are dependent on the motors, while the pitch and yaw depend on the ailevons.
+In FW mode, yaw depends on differential thrust from the motors, while roll and pitch depend on the ailevons.
+
+For airframe specific documentation and build instructions see [VTOL Framebuilds](../frames_vtol/README.md).
+
+## Firmware & Basic Settings
+
+1. Run *QGroundControl*
+2. Flash the firmware for your current release or master (PX4 `main` branch build).
+3. In the [Frame setup](../config/airframe.md) section select the appropriate VTOL airframe.
+   At time of writing, this must be the [Generic VTOL Tailsitter](../airframes/airframe_reference.md#vtol-tailsitter).
+
+
+### Flight / Transition Mode Switch
+
+You should assign a switch on your RC controller for changing between the multicopter and fixed wing modes.
+
+:::note
+While PX4 allows flight without an RC controller, you must have one when tuning/configuring a new airframe.
+:::
+
+This is done in [Flight Mode](../config/flight_mode.md) configuration, where you [assign flight modes and other functions](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set) to switches on your RC controller.
+The switch can also be assigned using the parameter [RC_MAP_TRANS_SW](../advanced_config/parameter_reference.md#RC_MAP_TRANS_SW).
+
+The switch in the off-position means that you are flying in multicopter mode.
+
+## Duo Tailsitter MC Mode Tuning Guide
+
+MC-mode tuning is dificult because some axes depend on control surfaces, while others depend on the motors.
+
+:::warning
+Always disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) when tuning a vehicle.
+:::
+
+As a sanity check, we start off by setting the following gain parameters:
+
+- Roll rate control:
```

Comment:
Cross posted comment from @sfuhrer 

--- 

Actually MC_PITCHRATE_X and MC_YAWRATE_X params shouldn't have an effect on the tuning of a duo tailsitter. In hover mode it uses the MC controller for the axes actuated by motors (only roll), while it uses the fixed-wing controller for pitch and yaw. So you normally tune:

For hover yaw:
FW_RR_P
FW_RR_I
FW_RR_FF (I recommend to keep this low, or even at 0)
and for hover pitch:
FW_PR_P
FW_P_I
FW_PR_FF

The convention is that control surfaces get controlled through the FW controller, and that for the FW controller the body axes are named as they are oriented in fixed-wing flight.
I acknowledge this is insanely confusing, and thus appreciate it even more that you started with this tuning guide!

Reply:
Wait what?
But I did see a change in the hover pitch and yaw response when I changed the MC gains!?!?!?

Reply:
[Is this what you are talking about? ](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/fw_rate_control/FixedwingRateControl.cpp#L245C14-L245C14)

Line 245 specifically.

Reply:
Okay so recalling the previous tests, I realised that altering the MC Attitude controller gains had an effect and not the MC Attitude Rate controller gains. I think this makes a little sense now.

Reply:
Yes the attitude gains will have an effect, but not the rate gains :+1:

Reply:
@HarshilNaikk I assume you will make the needed changes to these docs.

Reply:
I've added these changes.